networking/cidr-range-definitions.adoc

```diff
@@ -35,12 +43,37 @@ endif::[]
 
 [id="service-cidr-description"]
 == Service CIDR
-In the Service CIDR field, you must specify the IP address range for services. It is recommended, but not required, that the address block is the same between clusters. This will not create IP address conflicts. The range must be large enough to accommodate your workload. The address block must not overlap with any external service accessed from within the cluster. The default is `172.30.0.0/16`.
+In the Service CIDR field, you must specify the IP address range for services. 
+ifdef::openshift-rosa,openshift-dedicated[]
+It is recommended, but not required, that the address block is the same between clusters. This will not create IP address conflicts. 
+endif::openshift-rosa,openshift-dedicated[]
+The range must be large enough to accommodate your workload. The address block must not overlap with any external service accessed from within the cluster. The default is `172.30.0.0/16`.
 
 [id="pod-cidr-description"]
 == Pod CIDR
-In the pod CIDR field, you must specify the IP address range for pods. It is recommended, but not required, that the address block is the same between clusters. This will not create IP address conflicts. The range must be large enough to accommodate your workload. The address block must not overlap with any external service accessed from within the cluster. The default is `10.128.0.0/14`.
+In the pod CIDR field, you must specify the IP address range for pods. 
+
+ifdef::openshift-enterprise[]
+The pod CIDR is the same as the `clusterNetwork` CIDR and the cluster CIDR. 
+endif::openshift-enterprise[]
+ifdef::openshift-rosa,openshift-dedicated[]
+It is recommended, but not required, that the address block is the same between clusters. This will not create IP address conflicts. 
+endif::openshift-rosa,openshift-dedicated[]
+The range must be large enough to accommodate your workload. The address block must not overlap with any external service accessed from within the cluster. The default is `10.128.0.0/14`. You can expand the range after cluster installation.
+
+ifdef::openshift-enterprise[]
+[role="_additional-resources"]
+.Additional resources
+* xref:../networking/cluster-network-operator.adoc#nw-operator-cr-cno-object_cluster-network-operator[Cluster Network Operator Configuration] * xref:../networking/configuring-cluster-network-range.adoc#configuring-cluster-network-range[Configuring the cluster network range]
+endif::openshift-enterprise[]
```

Comment:
Strangely, this content looks right in the ROSA preview but not enterprise (caps below). Maybe hte preview bot didn't pick up a commit update?
![image](https://github.com/openshift/openshift-docs/assets/61474374/06da78e2-5de7-4591-9f93-da0b7da6b5e7)
![image](https://github.com/openshift/openshift-docs/assets/61474374/59da4359-dd17-493d-8e22-54e09c722a5b)

Reply:
LGTM :)
![image](https://github.com/openshift/openshift-docs/assets/61474374/7bc84ab6-8af8-4494-8020-fa8a98d7be8b)

Reply:
OK this is behaving now, pasting comps for giggles. Thanks @jeana-redhat for the halp here :) 
<img width="790" alt="Screenshot 2024-01-12 at 4 25 14 PM" src="https://github.com/openshift/openshift-docs/assets/77501157/c76c7786-46d0-4932-953d-d0351c1a4816">
<img width="777" alt="Screenshot 2024-01-12 at 4 25 50 PM" src="https://github.com/openshift/openshift-docs/assets/77501157/e3d52122-f28e-41db-a227-b7139513823c">